simulation/navigator_gazebo/models/wamv/wamv.sdf

```diff
@@ -143,11 +152,11 @@
         <plugin name="down_camera_controller" filename="libgazebo_ros_camera.so">
           <alwaysOn>true</alwaysOn>
           <updateRate>30</updateRate>
-          <cameraName>down/down</cameraName>
+          <cameraName>down</cameraName>
           <imageTopicName>image_raw</imageTopicName>
           <cameraInfoTopicName>camera_info</cameraInfoTopicName>
-          <frameName>down_down_cam</frameName>
-          <hackBaseline>0.89</hackBaseline>
+          <frameName>down_cam</frameName>
```

Comment:
If we're going to use sensible convention with this camera, why not move away from the convention used on the right camera (e.g. "/right/right")